pkgs/servers/home-assistant/custom-components/alarmo/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, fetchFromGitHub, buildHomeAssistantComponent }:
+
+buildHomeAssistantComponent rec {
+  owner = "nielsfaber";
+  domain = "alarmo";
+  version = "1.10.4";
+
+  src = fetchFromGitHub {
+    owner = "nielsfaber";
+    repo = "alarmo";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/hNzGPckLHUX0mrBF3ugAXstrOc1mWdati+nRJCwldc=";
+  };
+
```

Comment:
Delete everything in `custom_components/alarmo/frontend` except for the `dist` folder. Currently, we're shipping a lot of typescript source files.

Reply:
I did this in the `postInstall` hook - let me know if there's a better way.